data/data/powervs/iaas/power-iaas.tf

```diff
@@ -0,0 +1,22 @@
+provider "ibm" {
+ ibmcloud_api_key = var.ibmcloud_api_key
+}
+
+data "ibm_resource_group" "group" { 
+  name = var.cloud_resource_group
+}
+
+resource "ibm_resource_instance" "resource_instance" { 
+  name     = var.service_instance_name
+  service  = "power-iaas"
+  plan     = var.cloud_instance_id
```

Comment:
`cloud_instance_id` is what I was using to pass the service ID to the bootstrap module for node creation. Looking at our slack conversation and the docs [0] I think you were using it to pass in a proper value for `plan`. Really this should be using a new var and `cloud_instance_id` should be treated as something we use for dev purposes that we remove once the time comes


[0] https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/resource_instance

Reply:
hopefully i'm not muddling things here, but this is what we talked about earlier, and we can just set plan to `power-virtual-server-group` as per the embedded catalog offering info for power vs (i.e. `ibmcloud catalog service power-iaas`)